app/gateway/2.6.x/install-and-run/helm.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Install on Kubernetes with Helm
+---
+
+This page explains how to install {{site.base_gateway}} with {{site.kic_product_name}} using Helm. The Enterprise deployment includes a Postgres sub-chart provided by Bitnami. The documentation on installing with a [flat Kubernetes manifest](/gateway/{{page.kong_version}}/install-and-run/kubernetes) also explains how to install in DB-less mode for both Enterprise and OSS deployments.
+
+## Prerequisites
+
+- A Kubernetes cluster, v1.19 or later
+- `kubectl` v1.19 or later
+- (Enterprise only) A `license.json` file from Kong
+- Helm 3
+
+## Create namespace
+
+Create the namespace for {{site.base_gateway}} with {{site.kic_product_name}}. For example:
+
+```sh
+kubectl create namespace kong
+```
+
+## Set up Helm
+
+1.  Add the Kong charts repository:
+
+    ```sh
+    helm repo add kong https://charts.konghq.com
+    ```
+
+1.  Update Helm:
+
+    ```sh
+    helm repo update
+    ```
+
+## Create license secret
+{:.badge .enterprise}
+
+1.  Save your license file temporarily with the filename `license` (no file extension).
+
+1.  Run:
+
+    ```sh
+    kubectl create secret generic kong-enterprise-license --from-file=./license -n kong
+    ```
+
+## Create secret for RBAC superuser (recommended)
+{:.badge .enterprise}
+
+If you plan to use RBAC, you must create a secret for the superuser account password at this step in installation. You cannot create it later.
+
+1.  Create the RBAC account.
+
+1.  Create the secret:
+
+    ```sh
+    kubectl create secret generic kong-enterprise-superuser-password \
+    n kong \
+    --from-literal=password={YOUR_PASSWORD}
+    ```
+
+## Create secret for Session plugin
+{:.badge .enterprise}
+
+If you create an RBAC superuser and plan to work with Kong Manager or Dev Portal, you must also configure the Session plugin and store its config in a Kubernetes secret:
+
```

Comment:
I know that we had a short conversation where we decided to remove the portion of the doc that creates the sessions config file, but I forgot about the context. This was actually added to the doc later as users were constantly getting confused about the contents of the session file. This is necessary for RBAC to work. 

This step is generally skipped in other install topics because you don't need to set up secrets for RBAC the way you do for k8s, and  - but for this situation, how does the user know what these `admin_gui_session_conf` and `portal_session_conf` files are and where to find them if we don't provide more info?

(https://docs.konghq.com/enterprise/2.6.x/deployment/installation/kong-on-kubernetes/#step-5-prepare-the-sessions-plugin-for-kong-manager-and-dev-portal)

Reply:
yeah, will add in. Original idea was to link out to doc elsewhere, but I obviously forgot to add a note, or go figure out where else to link to.